package.json

```diff
@@ -26,19 +26,21 @@
     "react_ujs": "^2.6.0",
     "sass": "^1.35.2",
     "typescript": "^3.5.3",
-    "webpack": "^4.39.2",
-    "webpack-cli": "^3.3.7"
+    "webpack": "^4.46.0",
+    "webpack-cli": "^3.3.12"
   },
   "devDependencies": {
     "@babel/plugin-transform-modules-commonjs": "^7.5.0",
+    "@babel/plugin-transform-runtime": "^7.16.0",
     "@babel/preset-env": "^7.5.5",
     "babel-jest": "^24.9.0",
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-plugin-transform-decorators-legacy": "^1.3.5",
+    "dotenv-webpack": "^7.0.3",
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.14.0",
     "jest": "^26.6.3",
-    "webpack-dev-server": "^3.8.0"
+    "webpack-dev-server": "^3"
```

Comment:
is this actually a downgrade?

Reply:
So this is what came out when i ran the rails-webpack generator.
I checked the yarn.lock file, and there it's using version 3.11.2 now, so that is fine.

I'll change it to 3.11.2

Reply:
Fixed.